csi/moac/node_operator.ts

```diff
@@ -92,7 +92,7 @@ export class NodeOperator {
   ) {
     assert(registry);
     this.namespace = namespace;
-    this.workq = new Workq();
+    this.workq = new Workq('mayastornode');
```

Comment:
For the pool operator you have a commenting indicating the workq is for serializing pool operations. Do you want a similar comment here?

Reply:
Good point. Will add it 👍

Reply:
I just realised that there is already such comment at member definition. So that solves it.